### ISSUE TITLE
More anime providers

### ISSUE THREAD

**Issue by dulleh** [dulleh/akhyou#15]
Would improve the reliability of the app in being able to find certain anime.

Need HD sources.

Other languages too.

**Comment by dulleh** [comment]
**Suggestion: Masterani.me**  

This should be coming soon™ (soon as the api is up and working ^^) and I am in touch with the dev.

**Comment by dulleh** [comment]
**Suggestion: KissAnime.com**

Unlikely. They have that annoying loading page which makes it hard to scrape (because the site's contents are loaded through javascript).

**Comment by dulleh** [comment]
**Suggestion: chia-anime.tv**

**Comment by dulleh** [comment]
**Suggestion: rawranime.tv**

**Comment by dulleh** [comment]
**Suggestion: animeonhand.com**

**Comment by dulleh** [comment]
**Suggestion: animehaven.org**

**Comment by dulleh** [comment]
**Suggestion: ryuanime.com**

**Comment by dulleh** [comment]
**Suggestion: neko-san.fr**

**Comment by dulleh** [comment]
**Suggestion: hdvostfr.com**

**Comment by dulleh** [comment]
Official streamers too? Crunchy and the rest?

**Comment by dulleh** [comment]
There are so many this is actually kind of sad.

**Comment by wopian** [comment]
twist.moe ( devs interact with masterani.me devs and HB devs ;) )

**Comment by dulleh** [comment]
>twist.moe

 Oh yes, I've seen this around. I don't think I can just pick up their search feature however, so it comes after I've learnt more about search algorithms and what not (it's on the "things I need to learn" table!). May also go for a local database route with this site (and other tough to scrape sites _cough kissanime cough_) but that's also on the learning table.

**Comment by dulleh** [comment]
**Suggestion: anime1.com**

Seems easy but then getting the actual video is hard.

**Comment by Kushirin** [comment]
**Suggestion: underanime.net**
**Suggestion: jkanime.net**
**Suggestion: animeid.tv**
**Suggestion: animeflv.net**
**Suggestion: frozen-layer.com**

There are a lot, but it will be good have one at least. I can search for more if this are too hard to implement.

**Comment by wopian** [comment]
**Suggestion:** moetube.com

**Comment by hthuong09** [comment]
**Suggestion:** htvanime.com
This one was built on angular so I guess it won't be too hard to parse things.

About KissAnime. You could take a look here to see how the by pass the firewall of cloudflare.
[cloudflare-scrape](https://github.com/Anorov/cloudflare-scrape/)
I used this package for my small download crawler app. It 100% times working.

**Comment by wopian** [comment]
@dulleh, twist.Moe's entire library is on the index page, so yeah, a local dB for ones like this would be ideal.

**Comment by dulleh** [comment]
twist.moe isn't happening. I've spoken with one of the devs and the answer is no.

**Comment by yuukiw** [comment]
How about some German sites like proxer

**Comment by dulleh** [comment]
**Suggestion: myanimeon.com**

**Comment by dulleh** [comment]
^ accident :)

**Comment by dulleh** [comment]
**Suggestion: kisscartoon**

**Comment by Skipytr** [comment]
Suggestion: puzztv.com and turkanime.tv

**Comment by Chohdi-Kun** [comment]
Please Add Some Arabic Streamig Solutions..
Suggestions: Okanime.com / Add-Anime.net / Anyanime.com

**Comment by allenguarnes** [comment]
Suggestion: Since you already have KissAnime, can you add their other sites such as KissAsian? They're basically the same sites anyway, just different content. Although, they're not anime, but it would be great if you could add those too. :)

**Comment by LinkofHyrule89** [comment]
You might find this Kodi add-on for kiss anime very useful as it has ways to parse content, connect to the accounts including favorites and lists, and bypass cloud flair protection. https://github.com/HIGHWAY99/repository.thehighway/tree/master/repo/plugin.video.kissanime

**Comment by xepheratu** [comment]
Suggestion: moetube.net since kissanime keeps going down. Animebam is not loading subbed for me. Animeram isnt loading at all and we need more sites.

**Comment by lightxnoct** [comment]
You should add 9anime.to

**Comment by Atemu** [comment]
https://proxer.me

The website provides multiple streams on different websites.

**Comment by Eeveel** [comment]
Some French providers for VOSTFR pleases! like
www.full-anime.com/
www.neko-san.fr/
=)

**Comment by xdk78** [comment]
Polish, if which one will be easy to implement:
http://www.senpai.com.pl/anime/ 
http://shinden.pl/
http://anime-odcinki.pl/
http://on-anime.pl/
